flow2ml/__init__.py

```diff
@@ -1 +1,2 @@
-from .Flow import Flow
\ No newline at end of file
+from .Flow import Flow
+from .Auto_Results import Auto_Results
```

Comment:
Can you tell me the reason for adding this?

Reply:
Otherwise ... I cant Import it from flow2ml package.